test/client/http_service.test.js

```diff
@@ -67,8 +67,8 @@ describe('Executing http service functions', () => {
     });
 
     describe('And an available mock server', () => {
-      beforeAll(() => server.listen());
-      afterAll(() => server.close());
+      beforeEach(() => server.listen());
```

Comment:
@makesitgo, using beforeAll here causes the above test where no server is listening to actually pass but have an uncaught exception thrown